rubicon_ml/client/rubicon.py

```diff
@@ -38,24 +38,51 @@ def __init__(
         persistence: Optional[str] = "filesystem",
         root_dir: Optional[str] = None,
         auto_git_enabled: bool = False,
+        composite_config: Optional[List[Dict[str, Any]]] = None,
         **storage_options,
     ):
-        self.config = Config(persistence, root_dir, auto_git_enabled, **storage_options)
+        if composite_config is not None:
+            self.configs = [
+                Config(
+                    persistence=config["persistence"],
+                    root_dir=config["root_dir"],
+                    auto_git_enabled=auto_git_enabled,
+                )
```

Comment:
this also needs `**storage_options`

I think we should ultimately just make users pass a list of `Config`s if they want to use multiple backends - what do you think? we don't need to do it in this PR

Reply:
Good catch. Yes I was hoping to split this into two PRs. This one is mostly backwards compatible, and that change would require an API change.

Reply:
Also the fact that this didn't cause any unit tests to fail indicates a gap - let me address that before we merge here.

Reply:
> Yes I was hoping to split this into two PRs. This one is mostly backwards compatible, and that change would require an API change.

perfect, sounds good!

> Also the fact that this didn't cause any unit tests to fail indicates a gap - let me address that before we merge here.

thanks!